.baseline/findbugs/excludeFilter.xml

```diff
@@ -1,5 +1,9 @@
 <!-- See http://findbugs.sourceforge.net/manual/filter.html for syntax -->
 <FindBugsFilter>
+    <!-- ignore generated code -->
+    <Match>
+        <Class name="~.*\.*Immutable.*" />
```

Comment:
Why is FindBugs running on generated_src? Are you using an outdated version of processors? (to be clear, this is a gradle-processors bug if anything).

https://github.com/palantir/gradle-processors/blob/master/src/main/groovy/org/inferred/gradle/ProcessorsPlugin.groovy#L143